Rakefile

```diff
@@ -3,3 +3,23 @@ require "logstash/devutils/rake"
 task :default do
   system('rake -T')
 end
+
+task :gradle => "gradle.properties" do
```

Comment:
more importantly, `vendor` is a first-class citizen in our ci/jarvis-publish setups: https://github.com/logstash-plugins/logstash-output-elasticsearch/blob/master/ci/build.sh#L19